src/azure.rs

```diff
@@ -253,6 +253,16 @@ pub struct struct__AzColor {
 
 pub type AzColor = struct__AzColor;
 
+#[inline]
+impl PartialEq for AzColor {
+    fn eq(&self, other: &AzColor) -> bool {
+        if self.r != other.r || self.g != other.g || self.b != other.b || self.a != other.a {
```

Comment:
Why not just do:

self.r == other.r && self.g == other.g...

and skip the conditional block?